assets/js/components/dashboard-sharing/ModuleRecoveryAlert.js

```diff
@@ -195,20 +199,11 @@ export default function ModuleRecoveryAlert() {
 						'google-site-kit'
 					) }
 				</p>
-				<Button
-					onClick={ handleRecoverModules }
-					disabled={
-						recoveringModules ||
-						! Object.values( checkboxes ).some(
-							( checked ) => checked
-						)
-					}
-				>
-					{ __( 'Recover', 'google-site-kit' ) }
-				</Button>
-				<Spinner isSaving={ recoveringModules } />
 			</Fragment>
 		);
+		ctaLink = '#';
+		ctaLabel = __( 'Recover', 'google-site-kit' );
+		onCTAClick = handleRecoverModules;
```

Comment:
These seem to be duplicated unnecessarily. I think we can set the `ctaLink` and `ctaLabel` with these values below so long as there is an `onCTAClick` defined.

Reply:
Is this something we can simplify?